### DISCUSSION TITLE
Easier to change save destination

### DISCUSSION THREAD

**Discussion by Valdus92** [TriliumNext/Trilium#6062] [General]
I think it would be a great idea to easily move the save folder or move the destination to a folder that already holds files. I have many files in markdown and would like to add more in the same folder IF possible.

**Comment by eliandoran** [comment]
@Valdus92 , please note that TriliumNext does not store notes directly as files on the file system, but as a single SQLite database.

In this case, having both the SQLite database and markdown files in the same place might be a little bit strange.

But yes, generally I think it would be a good idea to be able to set the database location or even add support for opening multiple databases.
